pygments/lexers/prolog.py

```diff
@@ -226,16 +228,16 @@ class LogtalkLexer(RegexLexer):
             (r'\^', Operator),
             # Strings
             (r'"(\\\\|\\"|[^"])*"', String),
-            # Ponctuation
+            # Punctuation
             (r'[()\[\],.|]', Text),
             # Atoms
-            (r"[a-z]\w*", Text),
-            (r"'", String, 'quoted_atom'),
+            (r"[a-z][a-zA-Z0-9_]*", Text),
+            (r"[']", String, 'quoted_atom'),
         ],
 
         'quoted_atom': [
-            (r"''", String),
-            (r"'", String, '#pop'),
+            (r"['][']", String),
```

Comment:
Can you please shed some light on this? Why is a character class selector?

Reply:
No longer remember the motif. These changes are old (the original pull request was on BitBucket more than one year ago). But it works fine as I'm using this submitted version on a regular basis on a patched Pygments installation.

Reply:
Can you please give it a shot with `"''"` again? Other than that, should be ok to merge.

Reply:
Replace occurrences of `[']` with `'` and re-run the tests. No issues. Pushed the changes.